test/e2e/test_cluster_configs/windows/network_plugin/kubenet.json

```diff
@@ -1,5 +1,8 @@
 {
-	"env": {},
+	"env": {
+		"GINKGO_SKIP": "resolve DNS across windows and linux deployments|should expect containers to be recreated after node restart",
```

Comment:
Should we disable this with a ginkgo.Skip() in code if we detect Windows + kubenet?
That seems to be the more common pattern.
@jackfrancis what do you think?

Reply:
Right, we can add an additional condition here (AND NOT is kubenet):

https://github.com/Azure/aks-engine/blob/master/test/e2e/kubernetes/kubernetes_test.go#L1934

And then modify the skip message to say something like "we only run this test on Windows-enabled clusters using Azure CNI".

And yeah, then we can get rid of the updates to this test config file, as the test runner itself will always skip those non-working tests in a kubenet context.